app/bin/service/search.dart

```diff
@@ -69,9 +75,15 @@ Future _main(FrontendEntryMessage message) async {
         new SnapshotStorage(storageService, snapshotBucket));
 
     final ReceivePort taskReceivePort = new ReceivePort();
+    registerDartSdkIndex(new SimplePackageIndex.sdk(
+        urlPrefix: dartSdkMainUrl(toolEnvSdkVersion)));
     registerPackageIndex(new SimplePackageIndex());
     registerTaskSendPort(taskReceivePort.sendPort);
 
+    // don't block on SDK index updates, as it may take several minutes before
+    // the dartdoc service produces the required output.
+    _updateDartSdkIndex();
```

Comment:
I believe there is an optional hint about unawaited futures. This code also doesn't look like it creates a future. How about .then((_) {}) or something to show that we do wish to ignore this background future?